ui/components/component-library/button-link/button-link.scss

```diff
@@ -35,7 +35,7 @@
       opacity: 0.5;
     }
 
-    &.mm-button--disabled:hover {
+    &.mm-button-base--disabled:hover {
```

Comment:
If we are adding a --disabled class do we still need to use the base--disabled class?